gitops/platform/teams/backend/namespaces-staging-values.yaml

```diff
@@ -162,3 +348,24 @@ namespaces:
           - operator : In
             scopeName: PriorityClass
             values: ["high"]
+    networkPolicies:
+    - name: allow-orders
+      spec:
+        podSelector: {}
+        policyTypes:
+        - Ingress
+        ingress:
+        - from:
+          - namespaceSelector:
+              matchLabels:
+                kubernetes.io/metadata.name: orders
+    roleBindings:
```

Comment:
What is this role binding for ?

Reply:
For the chapter Namespace as a service, I created rolebinding to the team to give them read access only to their namespaces frontend team vs backend team